src/main/java/org/apache/sling/feature/maven/mojos/IncludeArtifactMojo.java

```diff
@@ -62,9 +113,6 @@
 
     @Override
     public void execute() throws MojoExecutionException, MojoFailureException {
-        if (includeArtifactClassifier == null) {
```

Comment:
If you allow "null", you need to handle it below in line 123

Reply:
Thanks for pointing this out. My IT tests did not cover the issue with already cached features which I fixed.
The reason to allow no classifier is that simple projects do not need as the group / artifact id and version are enough.